### ISSUE TITLE
[Programmatic Access – Make Code - Footer] Incorrect role is defined for 'Language & Version' button controls present under ‘Footer’ section.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3436]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note:** User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that incorrect role is defined for 'Language & Version' button controls or not.

**Actual Result:**
Incorrect role 'Link' is defined for 'Language & Version' button controls under ‘Footer’ section.

**Expected Result:**
Correct role should be define as 'Button' instead of link for 'Language & Version' button controls under ‘Footer’ section.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**MAS Reference:**
[MAS 4.1.2 – Name, Role, Value ](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
#URL:https://makecode.microbit.org/#

**Comment by Shalini3457** [comment]
![MAS4 1 2_Footer_Incorrect role is defined for Language and Version](https://user-images.githubusercontent.com/72871089/96090564-01c6ba00-0ee6-11eb-85e7-df1636998245.jpg)
